src/pages/Search/SearchFiltersCategoryPage.tsx

```diff
@@ -0,0 +1,96 @@
+import RadioListItem from '@components/SelectionList/RadioListItem';
+import useLocalize from '@hooks/useLocalize';
+import useThemeStyles from '@hooks/useThemeStyles';
+import localeCompare from '@libs/LocaleCompare';
+import * as SearchActions from '@userActions/Search';
+import ONYXKEYS from '@src/ONYXKEYS';
+
+function SearchFiltersStatusPage() {
+    const styles = useThemeStyles();
+    const {translate} = useLocalize();
+
+    const [searchAdvancedFiltersForm] = useOnyx(ONYXKEYS.FORMS.SEARCH_ADVANCED_FILTERS_FORM);
+    const activeItems = searchAdvancedFiltersForm?.category;
+    const policyId = searchAdvancedFiltersForm?.policyId ?? '-1';
+
+    const [callCategories] = useOnyx(ONYXKEYS.COLLECTION.POLICY_CATEGORIES);
+    const [singlePolicyCategories] = useOnyx(`${ONYXKEYS.COLLECTION.POLICY_CATEGORIES}${policyId}`);
+
+    let categories: string[] = [];
+    if (!singlePolicyCategories) {
+        categories = Object.values(callCategories ?? {})
+            .map((value) => Object.values(value ?? {}))
+            .map((value2) => value2.map((value3) => value3?.name))
+            .flat();
+    } else {
+        categories = Object.values(singlePolicyCategories ?? {}).map((value) => value.name);
+    }
+
+    const removeDuplicates = (array: string[]) => array.filter((value, index) => array.indexOf(value) === index);
+
+    const categoryNames = removeDuplicates(categories);
+
+    const categoryList = useMemo(
+        () =>
+            categoryNames
+                .sort((a, b) => localeCompare(a, b))
+                .map((name) => ({
+                    text: name,
+                    keyForList: name,
+                    isSelected: activeItems?.includes(name) ?? false,
+                    value: name,
+                })),
+        [categoryNames, activeItems],
+    );
+
+    const updateCategory = (values: Partial<FormOnyxValues<typeof ONYXKEYS.FORMS.SEARCH_ADVANCED_FILTERS_FORM>>) => {
+        SearchActions.updateAdvancedFilters(values);
+    };
+
+    return (
+        <ScreenWrapper
+            testID={SearchFiltersStatusPage.displayName}
+            shouldShowOfflineIndicatorInWideScreen
+            offlineIndicatorStyle={styles.mtAuto}
+        >
+            <FullPageNotFoundView shouldShow={false}>
+                <HeaderWithBackButton title={translate('common.category')} />
+                <View style={[styles.flex1]}>
+                    <SelectionList
+                        sections={[{data: categoryList}]}
+                        onSelectRow={(item) => {
+                            const isActive = activeItems?.includes(item.value);
+
+                            let newCategories;
+
+                            if (isActive) {
+                                newCategories = activeItems?.filter((category) => category !== item.value);
+                            } else {
+                                newCategories = [...(activeItems ?? []), item.value];
+                            }
```

Comment:
@luacmartins we need some help with deciding what kind of input UI to use for selection of multiple values.
Also how should it behave?
A. user selects multiple values and has to click some kind of green `Save` button update filters - just like on dates
B. user selects multiple values and on every click we update the form data, so that when there is no `Save` button

@289Adam289 can you please drop 1-2 screenshots in this thread so show how the picker looks right now?

Reply:
![image](https://github.com/user-attachments/assets/e1015bd7-fa15-4558-99f0-80e0a2c960e1)
![image](https://github.com/user-attachments/assets/743a99e1-c9e5-472f-9e38-65be5c256614)

Reply:
I think option A is my preference. @Expensify/design for input.

Reply:
Option A is my preference as well. Would it make sense to use our multi-select list pattern here instead of making what is usually a single-select list multi-select? Let me know if that doesn't make sense haha. Example of the multi-select list:
![CleanShot 2024-07-26 at 10 44 00@2x](https://github.com/user-attachments/assets/6d4406b5-f170-4226-bf80-8906209bd600)

Reply:
Yup, big time agree with Danny there that we should use the checkbox pattern we have.